### DISCUSSION TITLE
Silos - Wheat Storage + FEATURE

### DISCUSSION THREAD

**Discussion by Q4FOIL** [sunflower-land/sunflower-land#1264] [Art]
Last UPDATE :

![Schermata 2022-08-03 alle 12 26 12](https://user-images.githubusercontent.com/58088988/182586422-b259b9fa-0a64-4ab2-88b1-023aa09a1786.png)

![Schermata 2022-07-24 alle 20 36 10](https://user-images.githubusercontent.com/58088988/180661199-2c0b9c13-8c3c-46dc-8872-5a9b036d7002.png)

Hello, this 

![image](https://user-images.githubusercontent.com/58088988/180660786-e629c148-132f-4115-97ca-31caa8cc6995.png)

![Schermata 2022-07-24 alle 20 28 15](https://user-images.githubusercontent.com/58088988/180660929-eda014c3-aa75-450c-b1c7-bd17485fe035.png)

**FEATURE**

**_Grow 20% Wheat Faster_**

**Comment by Q4FOIL** [comment]
Best Place 

![Schermata 2022-07-29 alle 12 21 33](https://user-images.githubusercontent.com/58088988/181739411-6d1bef8e-fbb3-40c9-a976-3d1c7216ae71.png)

**Comment by Q4FOIL** [comment]
![Schermata 2022-07-29 alle 12 28 51](https://user-images.githubusercontent.com/58088988/181740566-76806358-538f-4c44-b3aa-781aa4b6ab29.png)

**Comment by IvanSkazkin** [comment]
**Too huge nft**

**Comment by JcEii** [comment]
The nft should be fitted in small sizes or pixels (e.g., 8x8, 12x12, 16x16, 18x18, etc.). Wheat Storage, this can be called a "silo" which is sized about 16x48. Silo is the storage of grains or wheat. I think I suggest changing the structure of it because it's too huge (change it into a silo) and adding some details about its purpose, like "if the player stores wheat in a silo or wheat storage, it will automatically feed the animals". This is my only suggestion, but your wheat storage is good and I'll support it. <3


Like this one...

C: Aveontrainer | DeviantArt
![image](https://user-images.githubusercontent.com/105348036/182021074-40f6cd63-8f85-4afd-86ee-6eff20b800fd.png)

![image](https://user-images.githubusercontent.com/105348036/182021130-0531664a-87a2-436a-b757-d07221ec75fc.png)

**Comment by Q4FOIL** [comment]
@JcEii 

This is my idea, but i think it's poor :P

16x48px
![image](https://user-images.githubusercontent.com/58088988/182073772-c3fba142-1de2-4f30-b64d-053fc49635ea.png)

32x96px
![image](https://user-images.githubusercontent.com/58088988/182074079-278fa579-7ca0-4048-b4fc-103dd7ec2388.png)

**Comment by Q4FOIL** [comment]
or recoloured

16x32px
![image](https://user-images.githubusercontent.com/58088988/182074784-bc178548-6011-437f-990b-853dceed645e.png)

32x96px
![image](https://user-images.githubusercontent.com/58088988/182074821-5ac78655-c159-42b3-bb9f-5ef2ea34aa21.png)

**Comment by Q4FOIL** [comment]
**FEATURE** 

_**The animals feed automatically**_

**Reply by JcEii** [reply]
Just like that, more fitted into the game. Support!! <3

**Reply by Q4FOIL** [reply]
I just follow your size, tell me the size i can do it Gioconda

❤️

**Comment by Q4FOIL** [comment]
@JcEii 

Another version
![image](https://user-images.githubusercontent.com/58088988/182586422-b259b9fa-0a64-4ab2-88b1-023aa09a1786.png)


![image](https://user-images.githubusercontent.com/58088988/182204522-26edc21d-4ab7-459b-aec1-bceaf4d75c41.png)
**34x106px**

![image](https://user-images.githubusercontent.com/58088988/182204642-ef6f2d43-45ad-474e-97f0-a63e320a1f74.png)
**64x212px**

**Comment by Q4FOIL** [comment]
@JcEii 

![image](https://user-images.githubusercontent.com/58088988/182586422-b259b9fa-0a64-4ab2-88b1-023aa09a1786.png)

**Reply by hujikol** [reply]
I think the 3d perspective of your silo is quite odd compared to the barn, the barn POV is from the up/ roof, and not by the side view like your silo does

**Reply by Q4FOIL** [reply]
Yes i understand, but it's very hard to have a good perspective, you have some ideas about? ❤️ 
@hujikol

**Comment by Q4FOIL** [comment]
![image](https://user-images.githubusercontent.com/58088988/183909732-d01154be-d48e-4fbe-a513-4163c847eb07.png)

![image](https://user-images.githubusercontent.com/58088988/183909880-1c55473e-cce1-479e-974b-63a2d10999a9.png)


@hujikol

**Reply by hujikol** [reply]
i think it needs more volume on it, make the roof longer and the body shorter 
![image](https://user-images.githubusercontent.com/12151051/183913731-d273923d-907a-49e5-bc88-e812faea7e93.png)
something like that.. to make the perspective similar

**Reply by hujikol** [reply]
or you can also copy the model of the home.. the tallest one
![image](https://user-images.githubusercontent.com/12151051/183914017-fcbc9538-95f8-4561-8c6d-a354e2dd82fc.png)
![image](https://user-images.githubusercontent.com/12151051/183914145-35f2e676-5758-4cda-9b08-2760e287af92.png)

the roof perspective is quite good on this one

**Reply by hujikol** [reply]
> i think it needs more volume on it, make the roof longer and the body shorter ![image](https://user-images.githubusercontent.com/12151051/183913731-d273923d-907a-49e5-bc88-e812faea7e93.png) something like that.. to make the perspective similar

also, we need to replace those haystacks of wheat to place the silos
![image](https://user-images.githubusercontent.com/12151051/183916267-c49a6624-e359-4d8e-841c-6ac5f2cb8f6a.png)

something like that

**Reply by hujikol** [reply]
last opinion form me, make the silos color more saturated, i think adding contrast to it also a good option by changing to blue like the coop or at the town blue-red complement
![image](https://user-images.githubusercontent.com/12151051/183917531-459afa30-81cc-4b5c-978e-f0e79357895b.png)

**Comment by Q4FOIL** [comment]
@hujikol 

![image](https://user-images.githubusercontent.com/58088988/183996027-3b130302-581d-4532-9976-4638d4ac2d69.png)     ![image](https://user-images.githubusercontent.com/58088988/184023086-36f519f2-3b37-4824-af9a-1e746fa256be.png)


![image](https://user-images.githubusercontent.com/58088988/183996073-68e12e69-cc3e-4efa-87d8-7455cc61df09.png)   ![image](https://user-images.githubusercontent.com/58088988/184023113-31b29ef6-3f43-4dc5-9000-975817262bd0.png)

**Reply by hujikol** [reply]
love it ❤️ , great work of art! do you mind putting it on the farm? want to see if it blends perfectly...

until then we need the main dev to see this NFT feature 👀

**Comment by Q4FOIL** [comment]
@hujikol 

![Schermata 2022-08-11 alle 08 07 08](https://user-images.githubusercontent.com/58088988/184073520-63097955-ea87-457e-aa4a-6efa28b5c4ba.png)

❤️ 


The boss it's @adamhannigan  😄

**Comment by Q4FOIL** [comment]
@hujikol 

![Schermata 2022-08-11 alle 09 53 36](https://user-images.githubusercontent.com/58088988/184088471-152d7646-f653-4eae-b084-dbab12b92bd6.png)


![image](https://user-images.githubusercontent.com/58088988/184086984-191776e7-cc8d-4ab0-9c19-3dab5bfe68d4.png) | ![image](https://user-images.githubusercontent.com/58088988/184087118-e04faff2-6690-4841-b952-017bad4ce1fc.png)

**Reply by hujikol** [reply]
i think its to big 😆

**Reply by hujikol** [reply]
consider my suggestion in replacing the haystack at that bottom left, if you put that on the right side of the barn it will use the cow space though xD
